lib/ThirdParty/html-to-react/index.d.ts

```diff
@@ -1,2 +1,3 @@
-declare module "html-to-react";
-declare module "html-to-react/lib/utils";
+declare module "html-to-react/lib/utils" {
+  export * from "html-to-react/types/utils";
```

Comment:
@zoran995 was this what you meant with the comment yesterday about re-export of internal types?

I also updated this PR to make the html-to-react imports only import what they use, which I believe helps with tree-shaking.